### DISCUSSION TITLE
Can't get Apache Container to start..

### DISCUSSION THREAD

**Discussion by koni-ey** [nextcloud/all-in-one#1207] [Questions] [answered]
Hey all!

i spent half a day trying to get the nextcloud AIO to work on my Ubuntu V-Server, but it just doesnt seem to work on my setup. Whenever i finished the initial setup, all containers start except the Apache Container. In the Logs it says:

`run: loading initial config: loading new config: http app module: start: tcp: listening on :443: listen tcp :443: bind: permission denied`

I already tried manually opening the port with `sudo iptables -A INPUT -p tcp --dport 443 -j ACCEPT`. 
I also had problems with the domain validation, but i just skipped that with `SKIP_DOMAIN_VALIDATION=true`. 

The full command i run is: 
`sudo docker run -it --name nextcloud-aio-mastercontainer --restart always -p 8080:8080 -p 8443:8443 -p 80:80 --volume nextcloud_aio_mastercontainer:/var/docker/nextcloud
--volume /var/run/docker.sock:/var/run/docker.sock:ro --user root --env NEXTCLOUD_DATADIR="/var/docker/nextcloud/userdata" --sig-proxy=false --env SKIP_DOMAIN_VALIDATION=true nextcloud/all-in-one:latest`

Does anyone have an idea what else might be the problem?
Thanks in advance!

**Comment by szaimen** [comment]
a possible explanation would be that you are already running a diffferent process that uses port 443. Can you check if that is the case?

**Reply by koni-ey** [reply]
I checked, but no other process uses that port.. :/. 
I now went back to the "regular" nextcloud installation via plesk. 
Will try it again next time it upsets me :).
Thanks anyway and keep up the great work with this nice project!

**Comment by szaimen** [comment]
Is the docker daemon running as root user?

**Comment by szaimen** [comment]
Which Docker version do you have installed? 
I just found this after googling which should be present in docker v20.10 and higher:
https://github.com/moby/moby/pull/41030

Also, is this docker rootless or normal docker?

**Comment by szaimen** [comment]
Can you also check if docker is correctly installed by following https://docs.docker.com/engine/install/centos/?

**Comment by szaimen** [comment, marked as answer]
I most likely found the problem: https://github.com/docker-library/haproxy/issues/159#issuecomment-853242578

You need linux kernel 4.11 or higher in order to make it work.
